src/ReservoirPair.sol

```diff
@@ -55,4 +55,9 @@ abstract contract ReservoirPair is AssetManagedPair, OracleWriter, ReentrancyGua
         blockTimestampLast = blockTimestamp;
         emit Sync(reserve0, reserve1);
     }
+
+    function setMaxChangePerSecond(uint8 aChangePerSecond) external override onlyFactory {
+        require(0 <= aChangePerSecond && aChangePerSecond <= MAX_CHANGE_PER_SEC, "RP: INVALID_CHANGE_PER_SECOND");
+        allowedChangePerSecond = aChangePerSecond;
```

Comment:
should be `_allowedChangePerSecond` if its private/internal. That said, shouldnt this be public?

Reply:
I was thinking about this... if we make it public, it'd increase the gas cost of function lookups. 

The only good reason to have it is for admins like us to check that this variable is correctly set?

Reply:
Yeah, I think contract auditability is worth it because it lets users know about the parameters when they look at it on Etherscan. We can always revisit this during optimization time

Reply:
yes auditability is important. Changed the visibility of the variable to `public`

Reply:
It's pretty cheap to do given it's a setter variable. In general im not sure how I feel about events. I think eventually EVM LOG will be reworked, for now I suggest emitting an event to help 3rd party tooling that perhaps wants to monitor this stuff.

Reply:
okay implemented with test. Not emitting event at instantiation but at when changed by the factory.

Reply:
Why not at instantiation?

Reply:
okay so, I'm aware that it's the convention to emit events even for instantiation. e.g. `Ownable` emits `OwnershipTransferred` when the contract is created. 

But currently we don't emit events it for `swapFee` and `platformFee` on instantiation. Should we change that as well? If we use the method you're proposing below, we can just call `updateSwapFee` and `updatePlatformFee` in the constructor, which simplifies things.

Reply:
Yes we should change those as well in a separate issue

Reply:
okay created #109 to be addressed with a PR later